README.md

```diff
@@ -18,6 +18,7 @@
 ![.NET MAUI Weather App on all platforms](Assets/maui-weather-hero-sm.png)
 
 ## Current News
+* November 13, 2024 - [What's new in .NET MAUI for .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10)
```

Comment:
Since the exact date is not known, can you make the date be just November 2025

Reply:
We do know the exact date? .NET Conf is going to be November 11-13, 2025